stable/aws-node-termination-handler/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: aws-node-termination-handler
 description: A Helm chart for the AWS Node Termination Handler
-version: 0.7.2
+version: 0.7.4
```

Comment:
go ahead and set this to 0.7.3. I'm going to include this, the security context patch, and the tolerate-all-taints PRs into a single Chart release for NTH, and then publish the Helm release for eks-charts.